### ISSUE TITLE
Ollama "[object Object]" text left in input field

### ISSUE THREAD

**Issue by holdengand** [langflow-ai/langflow#3102]
### Bug Description

Seems like some text got left in or inserted into the Ollama component when issues were fixed in it recently. New Ollama components have "[object Object]" text in the input field that throws an error:

Error while updating the Component

    1 validation error for MessageInput value Value error, Invalid value type <class 'dict'> [type=value_error, input_value={'text_key': 'text', 'dat...4bf6-81e5-dfd9b1c0dc48'}, input_type=dict] For further information visit https://errors.pydantic.dev/2.8/v/value_error

And models can't be refreshed or added until that text is removed. The component seems to work fine once that text is removed from the input field.

### Reproduction

Just pull a new Ollama model component from the sidebar and the error pops up immediately.

And the error pops up again when you try and refresh the models list.

### Expected behavior

There should be no error when loading the component, and the 'input' field should be empty on loading a new component.

### Operating System

windows 10

### Langflow Version

1.0.14

### Python Version

None

### Screenshot

![image](https://github.com/user-attachments/assets/cadfc979-4777-4649-8fff-eae79ae4584b)

**Comment by holdengand** [comment]
Ok, now I don't seem to have this problem anymore. I didn't do or change anything. Last night every time I created a new Ollama component the input had [object Object] in it by default, new or old flows. now its gone, no update or anything. Was this just on my end? What could have cause me to have had this issue before but its gone now?

**Comment by saikolasani** [comment]
Hi @holdengand, thank you for bringing this issue to our attention. It appears that the problem may have originated from a previous version, where some information was cached and persisted across multiple versions. We are noticing this happen more, so working on fixing this on our end. This could have caused an issue from an earlier version to continue affecting the current one. We're glad to hear that it's working now. Please let us know if something pops up again, but the component should work going forward.

**Comment by holdengand** [comment]
I suppose I forgot to mention, it was late when I originally posted this, that this issue popped up right after i had run the update from the previous version. It was the first thing i checked as I was one of the people who had mentioned an issue with the Ollama component that not working with the prompt component, which I think was something conflicting with a langchain version or something?

Regardless, I'm not sure if you mean this "[object Object]" text was cached on my end; because I can't imagine how it would have gotten there as I never input text into that field directly, and if its cached from another component supplying text into the box it would be odd text to have been what got saved, of all things. I believe I restarted langflow once and texted again a few times before I posed here and didn't do anything further to clear the cache or restart. I just went to bed leaving langflow open and tested in the morning and it just worked. I didn't even restart the browser or anything.

I'm glad not to think its a widespread issue, I'm not sure how many use the ollama component over the openAI one who would have even noticed, but its certainly got me confused.
